store/cachekv/store.go

```diff
@@ -102,34 +102,43 @@ func (store *Store) Write() {
 	// We need a copy of all of the keys.
 	// Not the best, but probably not a bottleneck depending.
 	keys := make([]string, 0, len(store.cache))
-
 	for key, dbValue := range store.cache {
 		if dbValue.dirty {
 			keys = append(keys, key)
 		}
 	}
 
-	sort.Strings(keys)
```

Comment:
See my comment: https://github.com/cosmos/cosmos-sdk/issues/10487#issuecomment-958414042
I don't think we can remove sorting.

Reply:
Roger that, thanks let me revert this change!

Reply:
Addressed, PTAL and thank you!